docs/usage/schema.md

```diff
@@ -146,3 +146,17 @@ Outputs:
 ```json
 {!.tmp_examples/schema_with_example.json!}
 ```
+
+For more fine-grained control, you can alternatively set `schema_extra` to a callable and post-process the generated schema.
```

Comment:
Maybe worth a note saying explicitly `schema_extra` allows the schema argument to be mutated, rather than returning a new altered dictionary. For notices to miss this potential stumbling block